packages/jasmine-parameterized/CONTRIBUTING.md

```diff
@@ -1,9 +1,11 @@
-# Contributing to Jasmine Parameterized
+# Contributing
 
-## Requirements
+## Getting Started
+
+Make sure you have the following requirements installed:
```

Comment:
nit: for some reason this reads weird to me. I think this is slightly better, or could just delete the word entirely
```suggestion
Make sure you have the following prerequisites installed:
```